examples/config-policy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": "0.1.0-dev",
+  "storage": {
+    "rootDirectory": "/tmp/zot"
+  },
+  "http": {
+    "address": "127.0.0.1",
+    "port": "8080",
+    "realm": "zot",
+    "auth": {
+      "htpasswd": {
+        "path": "test/data/htpasswd"
+      },
+      "failDelay": 1
+    },
+    "accessControl": {
+      "repositories": {
+        "repos1/repo": {
+          "policies": [
+            {
+              "users": ["alice", "bob"],
+              "actions": ["create", "read", "update", "delete"]
+            },
+            {
+              "users": ["mallory"],
+              "actions": ["create", "read"]
+            }
+          ],
+          "defaultPolicy": ["read"]
+        },
+        "repos2/repo": {
+          "policies": [
+            {
+              "users": ["bob"],
+              "actions": ["read", "create"]
+            },
+            {
+              "users": ["mallory"],
+              "actions": ["create", "read"]
+            }
+          ],
+          "defaultPolicy": ["read"]
+        }
+      },
+      "adminPolicy": {
+          "users": ["admin"],
+          "actions": ["create", "update", "delete"]
+        },
+      "defaultAdminPolicy": ["read"]

```

Comment:
do we really need this? I don't have a special case where I use defaultAdminPolicy, in the end it's just a union between actions and defaultAdminPolicy.
So why do we need it?